src/Flagged/Modal.jsx

```diff
@@ -0,0 +1,71 @@
+let {
+  onReport,
+  reportedAccountId,
+  contentModerationFlagValue,
+  ...forwardedProps
+} = props;
+
+const profileUrl = `#/${REPL_ACCOUNT}/widget/ProfilePage?accountId=${reportedAccountId}`;
+
+let getFlaggedAccountsList;
+
+// copied from Moderate.jsx so maybe it's wrong
+if (forwardedProps.open) {
+  getFlaggedAccountsList = Social.get(
```

Comment:
You don't need the existing accounts (or posts). Each user's self-moderation data is going into the moderate object with the accounts as keys. The `set` method merges keys into that object when it stores it in the contract.